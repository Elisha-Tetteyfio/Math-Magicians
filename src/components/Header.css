.heading {
  display: flex;
  justify-content: space-between;
}

.title {
  font-size: 50px;
  background-color: orange;
  border-radius: 75%;
}

.links {
  display: flex;
  list-style: none;
  justify-content: flex-end;
  padding: 0 50px;
}

.linkItem {
  padding: 5px 20px;
  color: #000;
  font-size: 22px;
  font-weight: 600;
  text-decoration: none;
}

.orange {
  color: orange;
  text-decoration: underline;
}
