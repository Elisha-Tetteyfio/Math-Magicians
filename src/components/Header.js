import React from 'react';
import './Header.css';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <>
    <div className="heading">
      <div className="title"> Math magicians </div>
      <ul className="links">
        <li>
          <NavLink to="/" exact className="linkItem" activeClassName="orange">Home</NavLink>
        </li>
        <li>
          <NavLink to="/calculator" className="linkItem" activeClassName="orange">Calculator</NavLink>
        </li>
        <li>
          <NavLink to="/quote" className="linkItem" activeClassName="orange">Quote</NavLink>
        </li>
      </ul>
    </div>
  </>
);

export default Header;
