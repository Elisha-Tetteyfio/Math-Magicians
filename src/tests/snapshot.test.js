import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';
import { StaticRouter } from 'react-router-dom';
// import * as styles from '../components/Header.css';

it('Test header', () => {
  const tree = renderer
    .create(
      <StaticRouter>
        <Header />
      </StaticRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
