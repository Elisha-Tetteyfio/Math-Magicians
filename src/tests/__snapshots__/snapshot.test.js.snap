// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test header 1`] = `
<div
  className="heading"
>
  <div
    className="title"
  >
     Math magicians 
  </div>
  <ul
    className="links"
  >
    <li>
      <a
        aria-current="page"
        className="linkItem orange1"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="linkItem"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="linkItem"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
